test/support/test_utils.ex

```diff
@@ -203,6 +203,17 @@ defmodule Logflare.TestUtils do
     }
   end
 
+  @doc """
```

Comment:
if this is only used in one location we don't need a separate function in utils for it, can simply be a private function in log_controller.ex module.

Reply:
Ah, my bad, I just realised you were talking about this one and not the ones in TestUtilsGrpc. I'll move it around.

Reply:
Good call, fixed!